exotations/exotica_core_task_maps/include/exotica_core_task_maps/joint_velocity_limit_constraint.h

```diff
@@ -0,0 +1,71 @@
+//
+// Copyright (c) 2019, Christopher E. Mower
+// All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are met:
+//
+//  * Redistributions of source code must retain the above copyright notice,
+//    this list of conditions and the following disclaimer.
+//  * Redistributions in binary form must reproduce the above copyright
+//    notice, this list of conditions and the following disclaimer in the
+//    documentation and/or other materials provided with the distribution.
+//  * Neither the name of  nor the names of its contributors may be used to
+//    endorse or promote products derived from this software without specific
+//    prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+// LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+// CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+// SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+// CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+// POSSIBILITY OF SUCH DAMAGE.
+//
+
+#ifndef EXOTICA_CORE_TASK_MAPS_JOINT_VELOCITY_LIMIT_CONSTRAINT_H_
+#define EXOTICA_CORE_TASK_MAPS_JOINT_VELOCITY_LIMIT_CONSTRAINT_H_
+
+#include <exotica_core/task_map.h>
+
+#include <exotica_core_task_maps/joint_velocity_limit_constraint_initializer.h>
+
+namespace exotica
+{
+/// \brief Joint velocity limit task map for non time-indexed problems.
+///
+/// JointVelocityLimitConstraint constrains the joint velocity within a specified percentage of the velocity limit.
+///
+class JointVelocityLimitConstraint : public TaskMap, public Instantiable<JointVelocityLimitConstraintInitializer>
+{
+public:
+    JointVelocityLimitConstraint();
```

Comment:
If default, no need for constructor and destructor (remove in both header and implementation).